### PR TITLE
Add how to send mail to Docker FAQ

### PR DESCRIPTION
**Pull request by xr09**
## Description

This PR adds the use case of sending mail notifications from a Docker deployment.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

File: source/docker/faq-wazuh-container.rst [hunk 1]
```diff
@@ -75,3 +75,42 @@ In order to preserve Elasticsearch data even after removing the Elasticsearch co
       - elastic-data:/usr/share/elasticsearch/data
 
 This will store elasticsearch data inside ``elastic-data`` volume in the Docker host's local file system.
+
+How can I send emails on a Docker deployment?
+---------------------------------------------
+
+Wazuh expects an SMTP relay in the config, usually on a VM you can install it on the same host (as described on this `blog post <https://wazuh.com/blog/how-to-send-email-notifications-with-wazuh/>`_), but in a container environment an individual service is recommended.
+
+The role of a SMTP relay is to forward the mail notifications from Wazuh to a valid mail server, in this example we're accepting unauthenticated mail from the manager and forwarding it to the SMTP service by using valid credentials.
```
Comment (davidfmiranda) [inline source/docker/faq-wazuh-container.rst:84]:
```suggestion
The role of an SMTP relay is to forward the mail notifications from Wazuh to a valid mail server, in this example we are accepting unauthenticated mail from the manager and forwarding it to the SMTP service by using valid credentials.
```

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @xr09, I've reviewed your PR and looks good to me. Nevertheless, I've made you a couple of suggestions.

Regards,

David

File: source/docker/faq-wazuh-container.rst (hunk 1 above)
Comment (xr09) [reply source/docker/faq-wazuh-container.rst]: Fixed in 83b25a20d

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**